webdriver/tests/document_handling/test_execute_script.py

```diff
@@ -0,0 +1,189 @@
+import pytest
+
+from tests.support.asserts import assert_same_element
+from tests.support.inline import inline
+from webdriver.client import element_key
+
+
+# Ported from Mozilla's marionette's test suite.
+
+# TODO:
+# * Return a window proxy
+# * Throw an exception during JS execution
+# * All error cases
+# * Probably more.
+
+def assert_elements_equal(session, expected, actual):
+    assert len(expected) == len(actual)
+    for e, a in zip(expected, actual):
+        e_id = e[element_key] if isinstance(e, dict) else e.json()[element_key]
+        a_id = a[element_key] if isinstance(a, dict) else a.json()[element_key]
+        assert e_id == a_id
```

Comment:
I would expect `session.execute_script` to return deserialised [`webdriver.client.Element`](https://github.com/w3c/web-platform-tests/blob/master/tools/webdriver/webdriver/client.py#L605) objects, but you really shouldn’t use the high-level `session.execute_script` API when testing the low-level details of the _Execute Script_ command.

Please take a look at examples from the other wdspec tests which define helpers such as this, which calls `session.transport.send` explicitly:

```py
def minimize(session):
    return session.transport.send("POST", "session/%s/window/minimize" % session.session_id)
```

If you could please update this test to not use `session.execute_script` that would be great. When you do you need to  make sure the response is a success and that the returned data structures match what you expect.